lesson14/main.js

```diff
@@ -71,6 +71,12 @@ const init = async () => {
   createNewList(val);
 };
 
+const getNumbers = () => {
+  const inputNumberId = document.getElementById("js-number")
```

Comment:
NITS：

I was curious about the naming.
Shouldn't it be `inputNumber`?

Reply:
I agree. I fixed it!